migration/migrator/migrations/system/20230815125306_add_git_safe_directory.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+
+def up(config):
+    DAEMON_USER = config.submitty_users['daemon_user']
+    RAINBOWGRADES_REPOSITORY = os.path.join(config.submitty['submitty_install_dir'], 'GIT_CHECKOUT','RainbowGrades')
+
+    gitconfig_path = f"/home/{DAEMON_USER}/.gitconfig"
+
+    gitconfig_content = """[safe]
+    directory = {}""".format(RAINBOWGRADES_REPOSITORY)
+
+    try:
+        with open(gitconfig_path, "w") as gitconfig_file:
+            gitconfig_file.write(gitconfig_content)
+        subprocess.run(["sudo", "chown", f"{DAEMON_USER}:{DAEMON_USER}", gitconfig_path], check=True)
```

Comment:
should you also have a similar chown in install_system.sh as well?

Reply:
Thanks for the insight, it is always better practice to be explicit than relying on previous code. Just made the change